source/includes/_resource.collections.md

```diff
@@ -210,7 +220,8 @@ vhx.collections.create({
       </td>
       <td>A set of key/value pairs that you can attach to a collection object. It can be useful for
 storing additional information about the collection in a structured format. Metadata keys must be
-strings. Metadata values can be strings, integers, or arrays. There are a few reserved keys that
+strings. Metadata values can be strings, integers, arrays, or images. An image metadata value
+must must be a url of an image (< 3MB) prefixed with the text <code>image_url:</code>. There are a few reserved keys that
```

Comment:
just thinking now that this paragraph is longer... 

for clarity, should we move the section about reserved keys right after we talk about keys? example with the 3 paragraphs:

```
A set of key/value pairs that you can attach to a collection object. It can be useful for storing additional information about the collection in a structured format.

Metadata keys must be strings. There are a few reserved keys that are auto-generated, which cannot be updated. These reserved keys .... 

Metadata values can be strings, integers, ... 
```